sdk/recoveryservices-siterecovery/Azure.ResourceManager.RecoveryServicesSiteRecovery/CHANGELOG.md

```diff
@@ -1,15 +1,16 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2022-10-01)
```

Comment:
The date here is for which day you expected to release the new version, not the api version

Reply:
I set it to `2023-01-10`